src/lib/cat_superhero.svelte

```diff
@@ -4,46 +4,107 @@
-  };
-
+  import type { RowData } from './types';
+  import AccordionItem from './accordion_item.svelte';
   const rows: RowData[] = [
     {
       title: 'Find',
-      text: 'Find a cat who needs your help'
+      text: 'Find a homeless cat who needs your help',
+      imageName: 'find.png',
+      imageAlt: '',
+      extraContent:
+        'Is it a cat or a kitten? Does it look injured, ill or hungry? Does it have an ear tip (a sign that it’s a community cat)? Is it friendly? Here are some tips to help you figure out if the kitty is homeless.',
+      resourceTexts: ['Tips'],
+      resourceLinks: ['https://fixnation.org/about-tnr/faqs/']
     },
     {
       title: 'Feed',
-      text: 'Feed the cat in the afternoon and evening'
+      text: 'Feed the cat in the afternoon or evening',
+      imageName: 'feed.png',
+      imageAlt: '',
+      extraContent:
+        'Get kitty on a regular feeding schedule, putting out dry food and water each afternoon or evening. This makes trapping a lot easier later! You can even make a <a href="https://fixnation.org/shelters-feeding-stations/">DIY feeding station.</a>',
+      resourceTexts: ['DIY Feeding Station'],
+      resourceLinks: ['https://fixnation.org/shelters-feeding-stations/']
     },
     {
-      title: 'Trap',
-      text: 'Set trap, wait, trap cat and rejoice!'
+      title: 'Prep',
+      text: 'Get a loaner trap from FixNation and book an appointment.',
+      imageName: 'prep.png',
+      imageAlt: '',
+      extraContent:
+        'Watch our <a href="https://www.youtube.com/watch?v=JgatHzlxnv0&t=2s">online training video</a> with easy step-by-step instructions, then borrow a special humane trap and cloth cover from our clinic. <a href="https://fixnation.org/about-tnr/free-feral-application/">Make an appointment online.</a> You’re all set!',
+      resourceTexts: ['Training Videos', 'Make Appointment'],
+      resourceLinks: [
+        'https://www.youtube.com/watch?v=JgatHzlxnv0&t=2s',
+        'https://fixnation.org/about-tnr/free-feral-application/'
+      ]
     },
     {
-      title: 'FixNation',
-      text: 'Get free resources from FixNation and make an appointment'
+      title: 'Trap',
+      text: 'Set trap, wait, trap cat and rejoice!',
+      imageName: 'trap.png',
+      imageAlt: '',
+      extraContent:
+        'Trap between 4 pm to 11.30 pm the evening before your appointment. Set food on some tin foil and place under the trap (not inside). Keep an eye on the trap – never leave it unattended – and wait for kitty to go inside. Be patient! <a href="https://www.youtube.com/watch?v=IarsSTLUdTw">Trap-Neuter-Return</a> is the first step towards a happier and healthier life for your community cat friend.',
+      resourceTexts: ['TNR'],
+      resourceLinks: ['https://www.youtube.com/watch?v=IarsSTLUdTw']
     },
     {
       title: 'Hold',
-      text: "Hold cat overnight until next day's surgery"
+      text: 'Hold the cat overnight until surgery the next day',
```

Comment:
```suggestion
      text: 'Hold the cat overnight until surgery the next day.',
```
To match the formatting on the Figma.